src/inputrules/ed-input-rules.js

```diff
@@ -1,22 +1,26 @@
 // Added as plugin
 
-import inputRules from 'prosemirror/dist/inputrules'
-import {buildInputRules} from 'prosemirror/dist/example-setup'
-import EdSchema from '../schema/ed-schema-full'
-import inputCode from './input-code'
-import iosDoubleSpace from './ios-double-space'
+import {inputRules, allInputRules} from 'prosemirror-inputrules'
+import {buildInputRules, buildKeymap} from 'prosemirror-example-setup'
+import {keymap} from 'prosemirror-keymap'
+import {baseKeymap} from 'prosemirror-commands'
+import EdSchema from '../schema/ed-schema'
+// import inputCode from './input-code'
+// import iosDoubleSpace from './ios-double-space'
 
 
 const edRules = buildInputRules(EdSchema)
-edRules.push(inputCode)
 
-const isIOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
-if (isIOS) {
-  edRules.push(iosDoubleSpace)
-}
+// edRules.push(inputCode)
 
-const rules = inputRules.allInputRules.concat(edRules)
+// const isIOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
```

Comment:
With pm15... https://discuss.prosemirror.net/t/release-0-15-0/519

> The main change in this release is that a bunch more things (typing, in-textblock backspacing) are left to the browser, and ProseMirror takes care not to re-render content when it doesn't have to. Which goes exactly against the original concept of ProseMirror ('never trust the browser, we'll do it ourselves'), but more and more issues came up where browser or platform features, such as spell-check, long-press to type character variants on OS, double-space for a period on iOS, were broken because the browser and/or the OS didn't realize what was going on.

> So for relatively uncontroversial functionality (there's not too much disagreement on what should happen to your document when you type a character), the editor now takes a hands-off approach.